content/news/2024-09-11-community-reflection-on-bevys-fourth-year/index.md

```diff
@@ -0,0 +1,64 @@
+
+<!-- more -->
+
+## What is Bevy?
+
+For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. Bevy is also free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. We have a [Quick Start Guide](/learn/quick-start/introduction). You can also check out [Bevy Assets](/assets/) for a library of community-developed plugins, crates, games, and learning resources.
+
+## Reflections
+
+We had plenty of responses this year! Here is a central collection of **Bevy's Fourth Birthday** posts:
+
+* **Carter Anderson ([@cart](https://github.com/cart))** <span class="people-role people-role-inline people-role-project-lead">Project Lead</span><span class="people-role people-role-inline people-role-maintainer">Maintainer</span>
+  * [Bevy's Fourth Birthday](https://bevyengine.org/news/bevys-fourth-birthday/)
+* **Alice Cecile ([@alice-i-cecile](https://github.com/alice-i-cecile))** <span class="people-role people-role-inline people-role-maintainer">Maintainer</span>
+  * [I landed my dream job making a Rust game engine. Now what?](https://bevyengine.org/news/dream-job/)
+* **Thierry Berger ([@Vrixyz](https://github.com/Vrixyz))**
+  * [Happy Bevybirthday](https://thierryberger.com/blog/bevy-4th-birthday/)
+* **BD103 ([@BD103](https://github.com/BD103))**
+  * [4 Years of Bevy](https://bd103.github.io/blog/2024-08-18-4-years-of-bevy)
+* **JMS55 ([@JMS55](https://github.com/JMS55))**
+  * [Bevy's Fourth Birthday - A Year of Meshlets](https://jms55.github.io/posts/2024-08-30-bevy-fourth-birthday/)
+* **Mikael Matveinen ([@mmatvein](https://github.com/mmatvein))**
+  * [Bevy's 4th birthday: Reflections in the context of our own project](https://www.reddit.com/r/bevy/comments/1f712hl/bevys_4th_birthday_reflections_in_the_context_of/)
+* **Peter Hayman ([@mrchantey](https://github.com/mrchantey))**
+  * [Bevy Turns Four](https://beetmash.com/blog/bevy-turns-four)
+* **doomy ([@piedoom](https://github.com/piedoom))**
+  * [birthday_system.system()](https://doomy.org/bevys-fourth-birthday/)
+* **Chris Biscardi ([@ChristopherBiscardi](https://github.com/ChristopherBiscardi))**
+  * [Bevy's Fourth Birthday](https://gist.github.com/ChristopherBiscardi/ab3fc92c3ba83e46cb5577bda2d287bc)
+
+## What are we working on right now?
+
+Here are some things we have coming up!
+
+* [Next Generation Scene / UI](https://github.com/bevyengine/bevy/discussions/14437): We're working on a brand new modern scene system that will make defining Bevy scenes in code, asset files, _and_ in the upcoming Bevy Editor _much_ nicer. This will also be the foundation for the Bevy Editor itself, which will be built _on top_ of the new Scene / UI system. In general this will be a massive step forward for Bevy's capabilities and user experience. Read the whole proposal for details!
```

Comment:
```suggestion
* [Next Generation Scene / UI](https://github.com/bevyengine/bevy/discussions/14437): We're working on a brand new modern scene system that will make defining Bevy scenes in code, asset files, _and_ the upcoming Bevy Editor _much_ nicer. This will also be the foundation for the Bevy Editor itself, which will be built _on top_ of the new Scene / UI system. In general this will be a massive step forward for Bevy's capabilities and user experience. Read the whole proposal for details!
```

The sentence flows better to me without the second `in`. Alternative is adding an `in` before `asset files` I think.